Tasks/IISWebAppManagementOnMachineGroup/Utility.ps1

```diff
@@ -29,6 +29,11 @@ function Manage-IISWebSite
         [string] $appPoolIdentity,
         [string] $appPoolUsername,
         [string] $appPoolPassword,
+
+        [string] $configureAuthentication,
+        [string] $basicAuthentication,
```

Comment:
no it can be multiple auth schemes, refer inetmgr screen shot

![image](https://cloud.githubusercontent.com/assets/11993108/24133329/235afac8-0e22-11e7-97b0-b8ec915b5623.png)